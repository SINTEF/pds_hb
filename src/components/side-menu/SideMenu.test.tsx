import React from 'react'
import renderer from 'react-test-renderer'
import { SideMenu } from '.'

it('renders without crashing', () => {
  const tree = renderer
    .create(<SideMenu menu={['Approve new users', 'test menu']} alert={6} />)
    .toJSON()
  expect(tree).toMatchSnapshot()
})
