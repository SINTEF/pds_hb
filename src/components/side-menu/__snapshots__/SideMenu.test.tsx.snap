// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing 1`] = `
<div
  className="container"
>
  <div
    className="home"
  >
    PDS Datahandbook
  </div>
  <div
    className="container"
  >
    <hr
      className="line"
    />
    <button
      className="button standard notify"
    >
      Approve new users
      <div
        className="notifysignal"
      >
        6
      </div>
    </button>
  </div>
  <div
    className="container"
  >
    <hr
      className="line"
    />
    <button
      className="button standard "
    >
      test menu
    </button>
  </div>
</div>
`;
